services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/api/rest/_resource_tracker.py

```diff
@@ -78,6 +81,38 @@ async def list_usage_services(
     )
 
 
+@router.get(
+    "/services/-/usages/presigned-link",
```

Comment:
question: why is it here GET presigned-link?
Does the user care if it's presigned?
in the webserver you export. why not re-use the same signature? POST :export? you even call a function called export right afterwards.

Reply:
will be deleted, it is not used (it was just for testing) :+1: